core/cloudflow-operator/src/test/kubernetes/shopping-cart-cr.json

```diff
@@ -0,0 +1,96 @@
+{
```

Comment:
I took this from the connected-cars example and modified it. Can be installed with kubectl cloudflow deploy src/test/kubernetes/shopping-cart-cr.json

Reply:
For discussion as well. Currently Kubeflow operator is a bit asymmetric. It uses external Flink and Spark operators, but is managing Akka streamlets internally. It would be nice to factor out Akka management as a separate operator - similar to Spark and Flink and keep Cloudflow one as a manager - outsourcing management of specific runtimes to specific operators. This will make the overall maintenance of Cloudflow application simpler and will enable simpler integration with additional runtimes (when/if we will need it)

Reply:
We have a design for exactly this, the downside of it is obviously that you need to run more operators, requiring more resources.

Reply:
At the same time, having said that, this might be a very nice way to do it, integrating between akka microservices and cloudflow in a more flexible way.

Reply:
Yea, I think this will a very nice separation of responsibilities and will make each operator simpler and more manageable

Reply:
See https://github.com/lightbend/cloudflow/issues/356 @blublinsky

Reply:
That sounds like a great addition